mephisto/scripts/local_db/gh_actions/auto_generate_provider.py

```diff
@@ -0,0 +1,32 @@
+from mdutils.mdutils import MdUtils
+from mephisto.client.cli_commands import get_wut_arguments
+from mephisto.operations.registry import (
+    get_valid_provider_types,
+)
+from mephisto.scripts.local_db.gh_actions.auto_generate_blueprint import (
+    create_blueprint_info,
+)
+
+
+def main():
+    provider_file = MdUtils(
+        file_name="../../../../docs/web/docs/reference/providers.md",
+    )
+    provider_file.new_header(level=1, title="Provider")
```

Comment:
All the other sibling pages are plural ("Architects", "Blueprints"). Can we call this "Providers" to stay consistent?